package.json

```diff
@@ -40,12 +37,6 @@
     "cli": "node --experimental-repl-await -r ./cli.js",
     "open-coverage": "open-cli coverage/lcov-report/index.html"
   },
-  "husky": {
```

Comment:
how is husky launched now? i saw the files above, i mean I don't understand how they are launched now

Reply:
If you have husky installed locally it runs when you do a commit.